interner/src/lib.rs

```diff
@@ -65,12 +61,6 @@ impl Deref for InternedString {
     }
 }
 
-impl Hash for InternedString {
-    fn hash<H: Hasher>(&self, state: &mut H) {
-        Hash::hash(&(self.inner as *const str), state)
```

Comment:
This is different from `Derive` d implementation and I want to leave it

Reply:
Ah, indeed! I didn't realize that raw pointers do not hash transparently.

Why is it necessary to not hash the same as a non-interned string?

Reply:
Just for speed up.
But, since I replaced most of `HashMap<InternedString, _>` with `Vec`, now we don't have to be too nervous about it.
To use `derive` is also OK if you preffer to it.

Reply:
If there is a good reason, I have no objections (other than that it should probably be commented...)

Reply:
I see

Reply:
I see